src/Tribe/Rewrite.php

```diff
@@ -506,6 +516,13 @@ protected function get_matcher_to_query_var_map() {
 	protected function get_localized_matchers() {
 		$localized_matchers = parent::get_localized_matchers();
 
+		// If possible add a `localized_slug` entry to each localized matcher to support multi-language.
+		array_walk( $localized_matchers, function ( array &$localized_matcher ) {
+			if ( isset( $localized_matcher['base'], $this->localized_bases[ $localized_matcher['base'] ] ) ) {
+				$localized_matcher['localized_slug'] = $this->localized_bases[ $localized_matcher['base'] ];
+			}
+		} );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 91168990167ae4d76d80a5b021f751c6</sub>